workflow/__init__.py

```diff
@@ -5,20 +5,24 @@
 '''
 import yaml
 import os
+from typing import Dict, Any
 
 # define defaults to be used during initialization
-DEFAULT_OUT_FOLDER = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-DEFAULT = '<value-here>'
+DEFAULT_OUT_FOLDER  = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
This is a different whitespace pattern than we've used in our other Python files.

Reply:
In some files I have lined up expressions so that they are easier to read, is this something that I should make consistent throughout the project? I only use it for global variables or defaults